tests/testthat/test_strip_html.r

```diff
@@ -0,0 +1,41 @@
+library(dplyr)
+library(testthat)
+
+test_that("all <> and values between are removed", {
+
+
+  cols <- c("What is your favorite color?", "Keep  value")
```

Comment:
Do you want to convert double-spaces that are the result of a tag removal into single spaces?  Maybe offering that as an option?

Reply:
Great idea! I added `trim_space` as an option.